app.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+var userName = prompt('What should I call you?');
+  alert('Okay, ' + userName + ', my name is Sean.');
```

Comment:
this is indented unnecessarily.  use white space to break up code, and indentation to keep indicate where things nest.